src/main/java/com/fasterxml/jackson/databind/deser/std/PrimitiveArrayDeserializers.java

```diff
@@ -112,15 +117,35 @@ protected T handleNonArray(JsonParser p, DeserializationContext ctxt) throws IOE
         boolean canWrap = (_unwrapSingle == Boolean.TRUE) ||
                 ((_unwrapSingle == null) &&
                         ctxt.isEnabled(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY));
+        
         if (canWrap) {
             return handleSingleElementUnwrapped(p, ctxt);
         }
+        // Handling the delimited list
+        else if(_format != null && 
+        		("".equals(_format.getPattern()) || "\\s*,\\s*".equalsIgnoreCase(_format.getPattern()))){
```

Comment:
I don't think it makes sense to use a "magical pattern" String to denote specific handling.
Instead we should decide on how `pattern` should be interpreted: I assume it would make sense to accept a standard JDK `Pattern` to denote separator to use. Alternative could be pattern for extraction.